_sass/blocks/case-studies/_intro.scss

```diff
@@ -34,12 +34,34 @@
 
 .case_studies_intro-select {
   max-width: 400px;
-  padding-top: $base-padding-lite;
   width: 100%;
 
+  label {
+    padding-bottom: $base-padding-lite;
+  }
+
   select {
     height: 40px;
   }
+
+  &.desktop {
+    display: none;
+  }
+
+  &.mobile {
+    display: block;
+  }
+
+  @include respond-to(medium-up) {
```

Comment:
Expected item on line 55 to appear before line 39. Rule sets should be ordered as follows: `@extends`, `@includes` without `@content`, properties, `@includes` with `@content`, nested rule sets